digital-twins/jsonld-contexts/environment.jsonld

```diff
@@ -0,0 +1,10 @@
+{
+    "@context": {
+        "depth": "https://smartdatamodels.org/dataModel.Environment/depth",
+        "co2": "https://smartdatamodels.org/dataModel.Environment/co2",
+        "pH": "https://smartdatamodels.org/dataModel.Environment/pH",
+        "salinity": "https://smartdatamodels.org/dataModel.Environment/salinity",
```

Comment:
can't really see where these 2 are used in TP. looks more like SMBT attributes.

also I would prefer having this file outside the DT directory (this is not DT per se, more related to an use-case)

Reply:
> can't really see where these 2 are used in TP. looks more like SMBT attributes.

It's probably coming from the Catalog module's data which reuse some SMBT data samples.

> also I would prefer having this file outside the DT directory (this is not DT per se, more related to an use-case)

If we are cherry-picking attributes for TP only, it makes sense to gather all the new contexts in the same place. This DT directory was just created for TP in the first place...

Reply:
Discussed during daily: OK to extract and keep this directory for core TP attributes